### PR TITLE
#163866138 add header and footer components

### PR DESCRIPTION
**Pull request by Proception**
#### What does this PR do?
It adds the header and footer component which also has the search and logo components

#### Description of Task to be completed?
- add header component
- add search component
- add logo component
- add footer component
- add snapshot tests for header, footer, logo, search, NotFound, Routes and Menu

#### How should this be manually tested?
- run `npm install`
- run `npm run test`
- run `npm run start:dev` and visit localhost:5050

#### Any background context you want to provide?
N/A

#### Screenshots (if appropriate)
screen shot 2019-02-12 at 8 37 10 am

#### What are the relevant pivotal tracker stories?
[163866138](https://www.pivotaltracker.com/story/show/163866138)

### CODE REVIEW

File: src/components/Menu.jsx [hunk 1]
```diff
@@ -1,15 +1,28 @@
...
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+        <Search/>
+         <ul class="nav navbar-nav mr-auto col-lg-2 float-right offset-1">
+            <li class="nav-item">
+              <NavLink to="/login" >Login</NavLink>
+            </li>
+            <li class="nav-item offset-1">
+              <NavLink to="/register" >Register</NavLink>
```
Comment (wombolo) [inline src/components/Menu.jsx:21]: Both Login and Signup features are on thesame page, I think they should have thesame route too.

File: src/index.jsx [hunk 2]
```diff
@@ -1,8 +1,10 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
-import 'bootstrap/dist/css/bootstrap.min.css';
```
Comment (julietezekwe) [inline src/index.jsx]: Why was this removed please?

File: src/components/Footer.jsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+export default class Footer extends Component {
+  render() {
+    return (
+      <footer>
+        <div>
+          <div className="float-left">
+            <li><h5>© 2019 All Rights Reserved</h5></li>
+          </div>
+          <div className="float-right">
+            <ul>
```
Comment (julietezekwe) [inline src/components/Footer.jsx:12]: Please use className instead of class

**Review by julietezekwe — CHANGES_REQUESTED** [review]
I dropped some comments.
Nice job Ben.

File: package.json [hunk 4]
```diff
@@ -68,5 +68,8 @@
     "webpack": "^4.29.3",
     "webpack-cli": "^3.2.3",
     "webpack-merge": "^4.2.1"
-  }
+  },
+  "jest": {
+       "testURL": "http://localhost:5050/"
```
Comment (abejide001) [inline package.json:73]: is there any specific reason for this 🤔

**Review by abejide001 — COMMENTED** [review]
I left a comment for you

File: package.json (hunk 4 above)
Comment (Proception) [reply package.json:73]: This is necessary to get the path during tests. It serves as the base path. It was used to run the test for the NotFound component

File: src/components/Menu.jsx (hunk 1 above)
Comment (Proception) [reply src/components/Menu.jsx:21]: I am not implementing login and signup, I dont think it will be in the context of my story to work on how it will be routed.

File: src/index.jsx (hunk 2 above)
Comment (Proception) [reply src/index.jsx]: That was an oversight will add it

File: src/components/Footer.jsx (hunk 3 above)
Comment (Proception) [reply src/components/Footer.jsx:12]: Done

File: package.json (hunk 4 above)
Comment (abejide001) [reply package.json:73]: okay 👍🏿

File: src/components/Search.jsx [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+    render() {
+        return (
+          <form className="form-inline my-2 my-lg-0 col-lg-7">
+            <input type="search" placeholder="Search" aria-label="Search for"/>
+          </form>
+        )
+      }
+}
+
+export default Search;
```
Comment (daniellamarr) [inline src/components/Search.jsx]: Please add a new line to the end of this file

**Review by daniellamarr — CHANGES_REQUESTED** [review]

File: src/components/Search.jsx (hunk 5 above)
Comment (Proception) [reply src/components/Search.jsx]: done
